libs/vm-modules/src/ml/utilities/mnist_utilities.cpp

```diff
@@ -0,0 +1,66 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "vm_modules/math/tensor.hpp"
+#include "vm_modules/ml/utilities/mnist_utilities.hpp"
+
+namespace fetch {
+
+namespace vm {
+class Module;
+}
+
+namespace vm_modules {
+
+namespace math {
+class VMTensor;
+}
+
+namespace ml {
+namespace utilities {
+
+fetch::vm::Ptr<fetch::vm_modules::math::VMTensor> load_mnist_images(
```

Comment:
a bit pedantic but you can drop some of the explicit namespacing in this file. everything is within the fetch scope so you don't need that. you can also drop the fetch::vm_modules:: namespacing